### PR TITLE
Provide more information on diff errors to help with troubleshooting

### PR DESCRIPTION
**Pull request by rvaidya**
Helps pinpoint issues when you get a list of errors from terraform plan (either from kustomize itself or from the dry run)

### CODE REVIEW

**Comment by pst** [comment]
Thanks for your contribution. I think it's a good initiative to improve the error output. But it should be done for all methods, not just ResourceDiff. Maybe it would be good to do `namespace/name` to make it more intuitive.

**Comment by rvaidya** [comment]
Updated - don't need the kind variable when we have gvr

File: kustomize/resource_kustomization.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func kustomizationResourceCreate(d *schema.ResourceData, m interface{}) error {
 	srcJSON := d.Get("manifest").(string)
 	u, err := parseJSON(srcJSON)
 	if err != nil {
-		return fmt.Errorf("ResourceCreate: %s", err)
+		return fmt.Errorf("ResourceCreate: JSON parse error: '%s' %s", srcJSON, err)
```
Comment (pst) [inline kustomize/resource_kustomization.go:51]: I'm not sure outputting all srcJSON is a good idea. The resource may be a secret and now credentials are in some CI/CD log. I have to admit, I don't even know what err includes in this case.

**Review by pst — COMMENTED** [review]
Thanks for the contribution. Overall this looks great. The only concern I have is with printing the entire JSON, which can be an issue if the resource is a secret.

Let me know what you think.

**Comment by pst** [comment]
To move this forward, I cherry-picked your changes into a new branch and removed the printing of the potentially sensitive json. See #67 for more details. Closing this PR.

**Pull request closed without merging**